.changeset/pre.json

```diff
@@ -11,6 +11,8 @@
     "eleven-coats-fix",
     "few-weeks-scream",
     "fuzzy-rabbits-nail",
+    "kind-suits-speak",
+    "long-candles-wave",
```

Comment:
This changeset needs updating to reflect #333 more accurately. Those edits should be done in a new PR, alongside @marionnegp's cta changes

Reply:
Please merge #489 first

Reply:
Both of these issues are now merged.